src/commands/app/undeploy.js

```diff
@@ -61,8 +66,12 @@ class Undeploy extends BaseCommand {
         const v = values[i]
         await this.undeployOneExt(k, v, flags, spinner)
         const assetUndeployLogEvent = getAuditLogEvent(flags, aioConfig.project, 'AB_APP_ASSETS_UNDEPLOYED')
-        if (assetUndeployLogEvent) {
-          await sendAuditLogs(cliDetails.accessToken, assetUndeployLogEvent, cliDetails.env)
+        if (assetUndeployLogEvent && cliDetails) {
+          try {
+            await sendAuditLogs(cliDetails.accessToken, assetUndeployLogEvent, cliDetails.env)
+          } catch (error) {
+            this.log(chalk.red(chalk.bold('Error: Audit Log Service Error: Failed to send audit log event for deployment.')))
```

Comment:
it is an error in sending, but to the cli is it an error? I would say it's more like a warning here. Just do a `this.warn` without all the `chalk` adornment.

Typo: should be `undeployment` (is that even a word?)